api/app/files/models.py

```diff
@@ -0,0 +1,15 @@
+import uuid
+from app import db
+
+class File(db.Model):
+	__tablename__ = "file"
+	id = db.Column(db.Integer, primary_key=True)
+	file_name = db.Column(db.String(50), unique=True, nullable=False)
+	mime_type = db.Column(db.String(20), nullable=False)
+	guid = db.Column(db.String(50), unique=True, nullable=False)
```

Comment:
I think drop the unique constraint here as well. It's highly unlikely that two guids will ever [collide](https://stackoverflow.com/a/39776).